SparkPost/2web2ui
# AC-1946 - remove form from manual entry form

### What Changed

- Removed <Form from manual entry form (date picker sub component)

### How To Test

- Visit pages with date pickers and make sure they work exactly as they do on staging/prod. 

Two pages that have date pickers:
`http://localhost:3100/lists/suppressions`
`http://localhost:3100/signals/analytics`

Files that import date picker:
```
src/pages/abTesting/components/fields/StatusFields.js:
  6: import { TextFieldWrapper, DatePickerWrapper } from 'src/components/reduxFormWrappers';

src/pages/blocklist/components/IncidentsCollection.js:
  9: import DatePicker from 'src/components/datePicker/DatePickerV2';

src/pages/reportBuilder/components/DateTimeSection/DateTimeSection.js:
  7: import DatePicker from 'src/components/datePicker/DatePickerV2';

src/pages/reports/messageEvents/components/MessageEventsSearch.js:
  16: import DatePicker from 'src/components/datePicker/DatePicker';

src/pages/signals/components/filters/DateFilter.js:
  4: import DatePicker from 'src/components/datePicker/DatePicker';

src/pages/suppressions/components/SuppressionSearch.js:
  8: import DatePicker from 'src/components/datePicker/DatePicker';
```

### To Do
